modules/ROOT/pages/setting-environment-variables.adoc

```diff
@@ -0,0 +1,24 @@
+= Setting Environment Variables
```

Comment:
Should we add something about the env variables for the mule that we use for tooling? Some people might think that adding those env variables to the run configuration would also affect the design time of the mule application, which won't be the case. @pablolagreca What do you think?

Reply:
Yes. Make sense. Please provide the content to be added to Diane